applications/EigenSolversApplication/python_scripts/dense_linear_solver_factory.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics as KM
+import KratosMultiphysics.EigenSolversApplication as KratosEigen
+from KratosMultiphysics import python_linear_solver_factory
+
+def ConstructSolver(configuration):
```

Comment:
I have some comments here but no time to review atm, can you please wait for my review?

Reply:
sure